### DISCUSSION TITLE
Docker images to run on CPU or GPU

### DISCUSSION THREAD

**Discussion by bartowski1182** [LostRuins/koboldcpp#251] [Show and tell]
I've made up some docker images for KoboldCPP, one for just CPU and one for both CPU and GPU (CPU only image is significantly smaller for anyone who isn't using a GPU)

Has been updated to 1.32.1

https://hub.docker.com/r/noneabove1182/koboldcpp-gpu

https://hub.docker.com/r/noneabove1182/koboldcpp

**Comment by LostRuins** [comment]
That's cool! Very nice.

**Reply by bartowski1182** [reply]
Updated to 1.32 :) https://hub.docker.com/r/noneabove1182/koboldcpp-gpu

**Reply by entmike** [reply]
Plans on updating it?  Newer versions include OpenAI-compatible endpoints which is nice.

**Reply by bartowski1182** [reply]
Ah sure! kinda let this one flounder a bit, i'll update it more regularly :)

**Reply by bartowski1182** [reply]
@entmike updated :) https://hub.docker.com/r/noneabove1182/koboldcpp-gpu/tags and future ones will be faster too!

https://github.com/noneabove1182/koboldcpp-docker

**Reply by LostRuins** [reply]
I linked it on the main repo page too
